### PR TITLE
Grid component done.

### PR DESCRIPTION
**Pull request by TJain02**
Screenshot (124)

### CODE REVIEW

**Review by masraniJay — COMMENTED** [review]
I think grids are used mostly when you need to play with two dimensions. It'd be great if you'd shown it here. Else, it's good to merge.

File: CSS Components/grids.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.grids-container-two ,.grids-container-three{
+    display: grid;
+    background-color:#eef2ff;
```
Comment (shubhmehra) [inline CSS Components/grids.css]: please use color from root

File: CSS Components/grids.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    padding: 1rem;
+    column-gap: 1rem;
+    margin: 1rem;
+}
+
+.gridItem{
+    background-color: rgba(255, 255, 255, 0.8);
```
Comment (shubhmehra) [inline CSS Components/grids.css]: same here

File: grids.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grids</title>
+    <link rel="stylesheet" href="/CSS Components/grids.css">
```
Comment (shubhmehra) [inline grids.html]: please avoid giving spaces to folders use underscore instead

**Review by shubhmehra — APPROVED** [review]
some small comments

File: grids.html (hunk 3 above)
Comment (TJain02) [reply grids.html]: Done

Comment (TJain02) [reply grids.html]: Done the changes.

File: CSS Components/grids.css (hunk 1 above)
Comment (TJain02) [reply CSS Components/grids.css]: done! Thank you!

**Pull request merged**